index.bs

```diff
@@ -470,6 +477,8 @@ Every {{Document}} has a <dfn for="Document">post-prerendering activation steps
 
       1. Let |doc| be |bc|'s [=active document=].
 
+      1. If |doc|'s [=Document/origin=] is the same as |predecessorBC|'s [=active document=]'s [=Document/origin=], then set |doc|'s [=Document/activation start time=] to the [=relative high resolution coarse time=] for |activation start time| and |doc|.
```

Comment:
When will the descendant `doc`'s `activation start time` be set? I see it set only for top-level browsing contexts..